README.rst

```diff
@@ -27,93 +27,9 @@ The CrashDB Shell (aka *Crash*) is an interactive `command-line interface`_
 Screenshot
 ==========
 
-.. image:: docs/query.png
+.. image:: https://raw.githubusercontent.com/crate/crash/master/docs/query.png
     :alt: A screenshot of Crash
 
-Prerequisites
```

Comment:
I think the README must still contain installation instructions.

Reply:
I do not want to duplicate the installation instructions. it is bound to happen that one of them is updated and the other one is missed

so in choosing between the README docs and the official website docs, I chose the official website docs as the preferred location

if we want to include installation help in the README, I would suggest that we add a new section pointing to the official docs for further help with installing

is that acceptable?

Reply:
I think this would be a bad user experience for developers. They look at the Github readme for instructions, not on the website. The instructions in the readme and on the website can differ, though. For the readme, the `pip install` command the the `curl` command to download the standalone version are totally sufficient.